app/Migrations/V1Dot11Dot22.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * AtroCore Software
+ *
+ * This source file is available under GNU General Public License version 3 (GPLv3).
+ * Full copyright and license information is available in LICENSE.txt, located in the root directory.
+ *
+ * @copyright  Copyright (c) AtroCore UG (https://www.atrocore.com)
+ * @license    GPLv3 (https://www.gnu.org/licenses/)
+ */
+
+declare(strict_types=1);
+
+namespace Pim\Migrations;
+
+use Atro\Core\Migration\Base;
+use Espo\Core\Exceptions\Error;
+use Espo\Core\Utils\Util;
+
+class V1Dot11Dot22 extends Base
+{
+    public function up(): void
+    {
+        $this->createIntermediateTable();
+
+        $this->createUiHandler();
```

Comment:
Remove this function and write dynamic logic. system will add the UI handler automatically.